tests/Api/Admin/AdminResetPasswordTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Tests\Api\Admin;
+
+use Sylius\Tests\Api\JsonApiTestCase;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+final class AdminResetPasswordTest extends JsonApiTestCase
+{
+    /** @test */
+    public function it_sends_administrator_password_reset_email(): void
```

Comment:
We do not check the dispatch of the email here, so either we should do this or adjust the name of the test.

And would be nice to add contract tests for both POST and PATCH endpoints.